src/common/options/global.yaml.in

```diff
@@ -301,6 +301,28 @@ options:
   - startup
   # default changed by common_preinit()
   with_legacy: true
+- name: finisher_batch_events
+  type: uint
+  level: advanced
+  desc: how many events can finisher process continuously before giving up CPU to allow other threads to run
+  default: 5
```

Comment:
This could be increase to 50 from my test without mds/ms/finisher/osdc debug logs enabled it works very well for me.

The debug logs will delay this much.